python/tests/integration/arcticdb/version_store/test_deletion.py

```diff
@@ -540,6 +540,37 @@ def test_delete_date_range_with_strings(version_store_factory, index_start):
     assert_frame_equal(vit.data, df)
 
 
+@pytest.mark.parametrize("prune_previous_versions", [True, False])
```

Comment:
Most of this test is the same as the one above it?

Reply:
Yes but I don't want to make `def test_delete_date_range_with_strings` more complicated. 
And I only need to test against lmdb